### DISCUSSION TITLE
is gurobi 12 supported?

### DISCUSSION THREAD

**Discussion by zhenyuefu** [metab0t/PyOptInterface#28] [General]
gurobi 12发布了，有支持的计划吗

**Comment by metab0t** [comment]
I am working on a new major release with proper support for nonlinear programming.

Support for Gurobi 12 (especially the new feature to add complex nonlinear constraint) will be added after the release.

**Comment by metab0t** [comment]
Gurobi 12 is supported in the latest 0.3.0 release.

The support for nonlinear function in Gurobi will be added afterward.

**Comment by metab0t** [comment]
The nonlinear programming functionality of Gurobi 12 is fully supported in v0.5.0 of PyOptInterface.
